src/index.js

```diff
@@ -24,19 +24,7 @@ export type Props = {
   className: string,
 }
 
-type State = {
-  style: { [key: string]: any },
-  type: string,
-  speed: number,
-  width: number,
-  height: number,
-  primaryColor: string,
-  secondaryColor: string,
-  preserveAspectRatio: string,
-  className: string,
-}
-
-class ContentLoader extends React.Component<Props, State> {
+class ContentLoader extends React.Component<Props> {
```

Comment:
Now we don't need of a class anymore, you should to change the `class` to `const`, such as: 
```js
const ContentLoader = () => {
    // Stuff
} 
```

Now we have a dumb component, it's smarter approach